tests/vmcheck/test-misc-2.sh

```diff
@@ -336,6 +339,13 @@ assert_not_file_has_content err.txt 'Updates and deployments are driven by TestD
 vm_rpmostree cleanup -p
 echo "ok upgrade when updates driver is registered"
 
+# Check that drivers are ignored if inactive even if registered
+vm_cmd systemctl stop test-driver.service
+vm_rpmostree upgrade 2>err.txt # notice we do not `--bypass-driver`
```

Comment:
Could also add e.g. `--unchanged-exit-77` here. That way we also fail the test if the upgrade didn't actually do an upgrade.